src/linearshrinkage.jl

```diff
@@ -103,7 +103,7 @@ LinearShrinkage(;
     corrected::Bool=false) = LinearShrinkage(target, shrinkage, corrected=corrected)
 
 """
-    cov(lse::LinearShrinkage, X, [weights::FrequencyWeights]; dims=1)
+    cov(lse::LinearShrinkage, X, [weights::FrequencyWeights]; dims=1, mean=nothing, drop_var0=false)
```

Comment:
I'd make `drop_var0` a field in `LinearShrinkage`, similarly to `corrected`. Everything else looks fine :+1:

Reply:
Excellent suggestion, thanks!